react-ui/src/components/ListItemLink.jsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * Row on vertically stacked list
```

Comment:
Specifically a row with an link to an internal page. The naming isn't clear on this.